src/status_im2/contexts/onboarding/common/navigation_bar/view.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im2.contexts.onboarding.common.navigation-bar.view
```

Comment:
I separated the navigation bar since it's a common component for all the onboarding screens, it receives a callback for the `info` button.